frontends/web/src/routes/account/utils.ts

```diff
@@ -100,6 +100,18 @@ export type TAccountsByKeystore = {
   accounts: IAccount[];
 };
 
+export const sortAccounts = (accounts: IAccount[]) => {
+  return Array.from(accounts).sort(
+    (ac1, ac2) => ac1.keystore.name.localeCompare(ac2.keystore.name)
+  );
+  // TODO: thinking we should use keystore.fingerprint account.code
+  /*
+  return Array.from(accounts).sort((ac1, ac2) => {
+    return `${ac1.keystore.fingerprint}${ac1.code}`.localeCompare(`${ac2.keystore.fingerprint}${ac2.code}`)
```

Comment:
I haven't tested it yet, but I think this should fix it.

WDYT about combining 
keystore.fingerprint + account.code ?

Reply:
actually the account.code already inclues the fingerprint, so it's not strictly needed.. should I keep as this is the intention?

Reply:
I think we can avoid adding the fingerprint explicitly, but no strong opinion